test/merchant_repository_test.rb

```diff
@@ -69,6 +69,16 @@ def test_find_all_merchants_by_updated_at
     assert_equal 0, result2.length
   end
 
+  def test_most_revenue_returns_top_x_merchant_instances_ranked_by_total_revenue
+    load_data
+    result_1 = @se.merchant_repository.most_revenue(1)[0].id
+    result_2 = @se.merchant_repository.most_revenue(3).count
+    result_3 = @se.merchant_repository.most_revenue(2).last.id
+    assert_equal 20, result_1
+    assert_equal  3, result_2
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.